science/rtl-sdr/Portfile

```diff
@@ -6,13 +6,13 @@ PortGroup           github 1.0
 
 name                rtl-sdr
 
-github.setup        osmocom rtl-sdr 5e73f90f1d85d8db2e583f3dbf1cff052d71d59b
-version             20220115-[string range ${github.version} 0 7]
-checksums           rmd160  b20663bafc564401df066039a4e83a5215d99093 \
-                    sha256  19ed112e1f7f425dd77c98d81d205ebe8ec7aa372e6a92bcb2a50cf6eca274fd \
-                    size    134207
+github.setup        osmocom rtl-sdr 2.0.2 v
```

Comment:
Please add the the `revision` line:
`revision 0`

Additionally, the new version (2.0.2) will not be recognized as an update when comparing to `20220115-.....`. So to make sure everyone who has the port installed already will be prompted to upgrade you should add the line:
`epoch 1`

Reply:
Latest version of this pr resolved these issues.